pom.xml

```diff
@@ -221,6 +221,45 @@
         </plugins>
       </build>
     </profile>
+      <profile>
+      <id>test-cpp</id>
+      <activation>
+        <property>
+          <name>BUILD_TESTS</name>
+          <value>ON</value>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>3.0.0</version>
```

Comment:
Nit: it would be nice to have a property like `antrun.version` or use dependencyMgmt to help keep the two antrun-plugin setups in sync, in case we need to update the plugin version in the future.